src/Pages/ProductDetail/Component/ImgPurchInfo.jsx

```diff
@@ -0,0 +1,203 @@
+import React, { Component } from 'react';
+import ImgSlide from './ImgSlide/ImgSlide';
+// import {plus, minus} from './function.js';
+import './ImgPurchInfo.scss';
+
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+class ImgPurchInfo extends Component {
+    constructor(){
+        super();
+        this.state={
+            count: 1,
+            sizeBtn: true,
+            showCalComp: false,
+        }
+    }
+
+    showComp = () =>{
+        if(this.state.showCalComp === true){
+            alert ("이미 선택한 옵션입니다.")
+        }
+
+        this.setState({
+            showCalComp: true,
+            sizeBtn: !this.state.sizeBtn
+        });
+    }
+
+    hideComp = () => {
+        this.setState({
+            showCalComp: false,
+            count: 1
+        })
+    }
+
+    handleClick = () =>{
+        this.setState({
+            sizeBtn: !this.state.sizeBtn,
+        })
+    }
+
+    countChange = (e) =>{
+        if(e.target.value>1){
+        this.setState({
+            count: e.target.value,
+        })}
+        else{
+            this.setState({
+                count: 1,
+            })
+        }
+    }
+
+    minus = () => {
+        if(this.state.count>1){
+        this.setState({
+            count: this.state.count-1,
+        })}
+        else{
+            this.setState({
+                count: 1,
+            })
+        }
```

Comment:
아래와 같이 삼항연산자를 이용하면 코드가 훨씬 간결해져요!
```suggestion
        this.setState({ count : this.state.count > 1 ? this.state.count-1 : 1 })
```